src/ui/actions/event-listeners.ts

```diff
@@ -510,14 +517,42 @@ function createReactEventMapper(eventType: SEARCHABLE_EVENT_TYPES) {
       }
 
       return res;
+    } else {
+      // TODO Semi-debug logging of the arguments. Hopefully can be removed soon.
```

Comment:
When is "soon"?  Under what conditions are we okay to remove this?

If we intend to remove this, would be nice to file a Linear issue for it. (Otherwise code like this tends to get overlooked).